app/routes/contact.tsx

```diff
@@ -99,35 +99,34 @@ export default function () {
         </>
       ) : (
         <Form method="post">
-          <Label htmlFor="service-fieldset">What do you need help with?</Label>
+          <Label htmlFor="service-fieldset">Which version of GCN?</Label>
```

Comment:
This isn't a complete sentence, but it ends with punctuation as if it was.

Reply:
I thought I had reverted this.  Sorry, will fix.